src/ukify/ukify.py

```diff
@@ -2486,6 +2484,12 @@ def finalize_options(opts: argparse.Namespace) -> None:
 
 def parse_args(args: Optional[list[str]] = None) -> argparse.Namespace:
     opts = create_parser().parse_args(args)
+
+    # argparse puts some unknown options in opts.positional. Make sure we don't
+    # try to interpret something that is an option as a positional argument.
+    if opts.positional and any((bad_opt := o).startswith('-') for o in opts.positional):
```

Comment:
If `opts.positional` is empty then `any` the generator will not iterate over anything, so this should be enough
```suggestion
    if any((bad_opt := o).startswith('-') for o in opts.positional):
```